apps/system/js/context_menu_view.js

```diff
@@ -125,6 +123,29 @@
     this.elements.list.appendChild(this.elements.cancel);
   };
 
+  ContextMenuView.prototype.focus = function() {
+    setTimeout(function() {
+      document.activeElement.blur();
+      this.elements.cancel.focus();
+    }.bind(this));
+  };
+
+  ContextMenuView.prototype._requestFocus = function() {
```

Comment:
I had add this function and move the 99 and 86, which are Service.request('focus');`, to here.